### PR TITLE
Remove xunit

### PR DESCRIPTION
**Pull request by lavalamp**
Can't merge this until https://github.com/kubernetes/kubernetes/pull/28191 is in.

### CODE REVIEW

**Comment by lavalamp** [comment]
oh, hm. @ixdy, is this going to break other PR builder jobs?

**Comment by ixdy** [comment]
yeah, now it probably will...

**Comment by ixdy** [comment]
One side effect of this is that the PR commit statuses will lose the strings like "Build finished. 124 tests run, 12 skipped, 1 failed." I'm not sure how folks feel about that?

**Comment by lavalamp** [comment]
gubernator should make the failures easy to see though.

**Comment by ixdy** [comment]
The `skip-if-no-test-files` and `status-add-test-results` options should be removed if we remove xunit.

File: jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml [hunk 1]
```diff
@@ -201,23 +201,6 @@
         - shell: JENKINS_BUILD_STARTED=true "${{WORKSPACE}}/_tmp/upload-to-gcs.sh"
         - shell: '{cmd}'
     publishers:
-        - xunit:
-            thresholds:
-                - failed:
-                    unstable: 0
```
Comment (ixdy) [inline jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml]: I wonder what happens if we unset this - will it make the plugin barf, or will it make the plugin parse test results but never fail?

**Comment by lavalamp** [comment]
@ixdy you mean just remove the lines entirely everywhere they appear?

File: jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml (hunk 1 above)
Comment (lavalamp) [inline jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml]: https://github.com/kubernetes/test-infra/pull/246

**Pull request closed without merging**